src/contract/output_labels.jl

```diff
@@ -10,10 +10,11 @@ function output_labels(
   return output_labels(f, alg, labels1, labels2)
 end
 
-function output_labels(f::typeof(contract), alg::Algorithm, labels1, labels2)
+function output_labels(f::typeof(contract), ::Algorithm, labels1, labels2)
   return output_labels(f, labels1, labels2)
 end
 
 function output_labels(::typeof(contract), labels1, labels2)
-  return Tuple(symdiff(labels1, labels2))
+  diff = symdiff(labels1, labels2)
+  return tuplemortar((Tuple(intersect(diff, labels1)), Tuple(intersect(diff, labels2))))
```

Comment:
```suggestion
  diff1 = Tuple(setdiff(labels1, labels2))
  diff2 = Tuple(setdiff(labels2, labels1))
  return tuplemortar(diff1, diff2)
```
I think that is a bit easier to reason about and requires fewer set operations.